test/index.test.ts

```diff
@@ -1,14 +1,17 @@
 import os from 'os';
 import { TempDir } from './util';
-import { cosmiconfig, Options, LoaderSync } from '../src';
-import { createExplorer } from '../src/createExplorer';
-import * as loaders from '../src/loaders';
+import { LoaderSync } from '../src';
+import * as createExplorer from '../src/createExplorer';
+import * as createExplorerSync from '../src/createExplorerSync';
+import { loaders } from '../src/loaders';
 
-// @ts-ignore
-const createExplorerMock: typeof createExplorer & jest.Mock = createExplorer;
+let createExplorerMock = jest.spyOn(createExplorer, 'createExplorer');
+let createExplorerSyncMock = jest.spyOn(
+  createExplorerSync,
+  'createExplorerSync',
+);
```

Comment:
Do we need to `spyOn` here if we're doing it in the `beforeEach` hook?

Reply:
For testing, no. I added it so typescript can infer the correct type. Maybe there is a better way to do it, but I am unsure how?